### DISCUSSION TITLE
Dead crew should not be seen as dead until found.

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5926] [Ideas and suggestions]
Until someone grabs/heals a corpse, I do not think they should be shown as dead on the crew manifest. This is so that traitors can more easily accomplish objectives unnoticed, in addition to immersion being preserved.

**Comment by ViktorShield** [comment]
Or if a MEDIC sees the corpse via health scanner

**Comment by Likvidus** [comment]
This is a good idea. In this case, most likely it will be necessary to change the healthbar system. Now it is very easy to understand whether a person is alive or dead without a detailed examination of the body, because you can just look at the nickname and healthbar.

**Comment by Hazmat616** [comment]
Nothing would be more chilling than sending out a dive team getting warbled messages and then silence. Not knowing if they will ever return. 10/10 need.

**Comment by Ivanzyfer** [comment]
It would be possible to add this as a separate setting for a high difficult level
